src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit');
+    const languageRadioOptions = document.querySelectorAll('.ez-extra-actions--edit [type="radio"]');
```

Comment:
I the prevent class would be added you can use selector: `doc.querySelector('.ez-extra-actions--edit.ez-extra-actions--prevent-show [type="radio"]')`